stackstorm.yml

```diff
@@ -13,6 +13,6 @@
     - StackStorm.st2chatops
     - StackStorm.st2smoketests
     - role: StackStorm.ewc
-      when: ewc_license is defined and ewc_license is not none and ewc_license | length > 0
+      when: ewc_license is defined and ewc_license is not none and ewc_license | length > 1
```

Comment:
@armab How does this work? Check has changed to be that the license key is longer than 1 char, but how does that relate to:
"In this PR I'm just disabling the EWC installation and testing for unstable repositories by setting the BWC_LICENSE_ENTERPRISE_UNSTABLE ENV variable to 0."

I am presuming that the setting of the environment variable to "0" is done separately, rather than under this PR. And then this PR is making use of it, to then NOT use the ewc role when set this way.

Reply:
Yes, these variables are part of the TravisCI and set manually in the settings (https://travis-ci.org/github/StackStorm/ansible-st2/settings).